src/adapters/postgres/user-adapter.ts

```
@@ -1288,8 +1288,15 @@ export class PostgresUserService implements IServicelocator {
       const userId = userDto.userId;
       const keycloakReqBody = { username, firstName, lastName, userId, email };
 
-      //Update userdetails on keycloak
-      if (username || firstName || lastName || email) {
+      const userData = userDto.userData;
+      const shouldSyncKeycloak =
+        (userData && Object.prototype.hasOwnProperty.call(userData, 'username')) ||
```

ESLint

[error] Replace `·Object.prototype.hasOwnProperty.call(userData,·'username'` with `⏎··········Object.prototype.hasOwnProperty.call(userData,·"username"`

(prettier/prettier)